package.json

```diff
@@ -1,53 +1,54 @@
 {
-  "name": "resin-register-device",
-  "version": "5.0.0",
-  "description": "Resin.io device registration utilities",
-  "main": "build/register.js",
-  "homepage": "https://github.com/resin-io-modules/resin-register-device",
-  "repository": {
-    "type": "git",
-    "url": "git://github.com/resin-io-modules/resin-register-device.git"
-  },
-  "keywords": [
-    "resin",
-    "device",
-    "register"
-  ],
-  "directories": {
-    "test": "tests"
-  },
-  "scripts": {
-    "build": "gulp build",
-    "lint": "resin-lint lib tests gulpfile.coffee",
-    "test-node": "gulp test",
-    "test-browser": "karma start",
-    "test": "npm run lint && npm run test-node && npm run test-browser",
-    "prepublish": "npm test && npm run build"
-  },
-  "author": "Juan Cruz Viotti <juan@resin.io>",
-  "license": "Apache-2.0",
-  "devDependencies": {
-    "chai": "^3.5.0",
-    "chai-as-promised": "^6.0.0",
-    "coffee-script": "~1.12.2",
-    "gulp": "^3.8.10",
-    "gulp-coffee": "^2.2.0",
-    "gulp-mocha": "^3.0.0",
-    "gulp-util": "^3.0.1",
-    "karma": "^1.3.0",
-    "lodash": "^4.0.0",
-    "mocha": "^3.0.0",
-    "resin-config-karma": "^1.0.4",
-    "resin-fetch-mock": "^1.0.1",
-    "resin-lint": "^1.4.0",
-    "resin-request": "^9.0.2",
-    "sinon": "^1.17.6"
-  },
-  "dependencies": {
-    "bluebird": "^3.0.0",
-    "randomstring": "^1.1.5"
-  },
-  "peerDependencies": {
-    "resin-request": "^9.0.2"
-  }
+	"name": "resin-register-device",
```

Comment:
This file should be space-indented as it's controlled by the `npm` cli; running something like `npm i`/`npm update` should reformat it automatically

Reply:
Oh man didn't even notice the entire file changing.